src/validation/custom.validation.ts

```diff
@@ -16,8 +16,8 @@ export const checkMaxFieldIfRadioButtonSelected = (selected: boolean, errMsg: st
   return true;
 };
 
-export const checkInvalidCharactersIfRadioButtonSelected = (selected: boolean, errMsg: string, value: string) => {
-  if (value && !(selected || value.match(VALID_CHARACTERS))) {
+export const checkInvalidCharactersIfRadioButtonSelected = (noSelected: boolean, errMsg: string, value: string) => {
+  if (noSelected && !value.match(VALID_CHARACTERS)) {
```

Comment:
I still don't know why this function isn't just implemented in a similar way to the `checkMaxFieldIfRadioButtonSelected` function above. Surely it's contract is the same - throw an error if radio button selected and invalid characters are found, return true otherwise...